docker-compose.yml

```diff
@@ -1,6 +1,8 @@
-app:
-  build: .
-  ports:
-    - '4000:4000'
-  volumes:
-    - .:/app
+version: "3"
```

Comment:
Turns out `docker-compose` files now have versions, and 3.x is the latest supported version.

I caused an error here in Codespaces by using a `3.7` version and instead set it to `3`, so it all seems fine now :man_shrugging: